modules/geoip-lite-2024-07-19/scripts/updatedb.js

```
@@ -0,0 +1,685 @@
+// fetches and converts maxmind lite databases
+
+'use strict';
```

Biome

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)